src/app/map/components/MapContainer.js

```diff
@@ -115,26 +228,48 @@ class MapContainer extends Component {
             enableMobileScroll(map);
             map.addControl(new mapboxgl.FullscreenControl());
             map.addControl(new mapboxgl.ScaleControl());
+
+            // Change the cursor to a pointer when the mouse is over the places layer.
+            map.on('mouseenter', 'trashes', () => {
+              map.getCanvas().style.cursor = 'pointer';
+            });
+
+            // Change it back to a pointer when it leaves.
+            map.on('mouseleave', 'trashes', () => {
+              map.getCanvas().style.cursor = '';
+            });
           }
         }
-        >
-          <Layer // Layer with trashes
-            type="symbol"
-            id="trashes"
-            layout={{ 'icon-image': 'trash', 'icon-allow-overlap': true }}
-            images={['trash', trashIcon]}
           >
-            {/* Add containers from state */}
-            { (containers.length > 0)
-              ? containers.map(elem => (
-                <Feature
-                  key={elem.id}
-                  coordinates={[elem.longitude, elem.latitude]}
-                />))
-              : null}
-          </Layer>
-        </Map>
-      </MapComponent>
+            <Layer
+              // Layer with trashes
+              type="symbol"
+              id="trashes"
+              layout={{ 'icon-image': 'trash', 'icon-allow-overlap': true }}
+              images={['trash', trashIcon]}
+            >
+              {/* Add containers from state */}
+              { (containers.length > 0)
+                ? containers.map(elem => (
+                  <Feature
+                    key={elem.id}
+                    coordinates={[elem.longitude, elem.latitude]}
+                    onClick={() => this.showInfo(elem.id, elem.longitude, elem.latitude)}
+                  />))
+                : null}
+            </Layer>
+            { (containers.length > 0 && selectedId != 0)
```

Comment:
Please change !== to !=. selectedId is defined as Number, so it has the same type as 0.